.env.example

```
@@ -1,4 +1,5 @@
 ALMA_API_MODE=live
 ALMA_LIVE_API_KEY=sk_live_xxxxxxxxxxxxxxxxxxxxxxxx
 ALMA_TEST_API_KEY=sk_test_xxxxxxxxxxxxxxxxxxxxxxxx
-SFCC_SITE_NAME=RefArchGlobal
\ No newline at end of file
+SFCC_SITE_NAME=RefArchGlobal
+TOGGLE_REFUND=off/on
```

```suggestion
TOGGLE_REFUND=off|on
```
`a|b` means this value should either `a` or `b`